nxdrive/engine/engine.py

```diff
@@ -682,6 +684,19 @@ def _create_local_watcher(self) -> LocalWatcher:
     def _get_db_file(self) -> Path:
         return self.manager.home / f"ndrive_{self.uid}.db"
 
+    def _disk_space(self) -> tuple:
+        Disk = namedtuple("Disk", "total free used local_sync_files")
+        folder = str(self.local_folder)
+
+        total = shutil.disk_usage(folder).total
```

Comment:
Instead of calling 3 times `shutil.dis_space()`, store the result and use it directly:
```python
space = shutil.disk_usage(folder)
return DiskUsage(
    total=space.total,
    free=space.free,
    used=space.used,
    synced=self.dao.get_global_size(),
)
```